model_info/arwiki.damaging.md

```diff
@@ -15,67 +15,67 @@ Model Information:
 	 - python_implementation: 'CPython'
 	 - python_revision: ''
 	 - python_version: '3.5.3'
-	 - release: '4.9.0-6-amd64'
+	 - release: '4.9.0-8-amd64'
 	
 	Statistics:
-	counts (n=18637):
+	counts (n=18611):
 		label        n         ~True    ~False
 		-------  -----  ---  -------  --------
-		True       340  -->       14       326
-		False    18297  -->       18     18279
+		True       340  -->        1       339
```

Comment:
This doesn't look good.  Aren't these the actual counts?  What happened to our damaging examples?

Reply:
There is the same number of damaging examples.  The model just (randomly) decided to build more strictly for the "predicted" threshold.

Reply:
Ah!  Thanks, I hadn't thought about this implication of the 50% threshold.  Maybe we should actually be more explicit about this sort of thing, and choose a default threshold e.g. 60% prediction for "true" which is designed to balance some operational outcome.  I'm not sure what target metric would make sense here.

It's crazy that there isn't a standard orientation for these tables.